app/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = yeoman.generators.Base.extend({
     this.directory('.', '.');
   },
 
+  generateYoRc: function() {
+    this.log('Generating .yo-rc.json');
+    this.config.save();
+  },
+
```

Comment:
What do you need `.yo-rc.json` for? I don't see any other usage of `this.config` in this patch.